store/src/java/com/zimbra/qa/unittest/TestImapDaemonNotifications.java

```diff
@@ -32,4 +34,23 @@ public void tearDown() throws ServiceException, DocumentException, ConfigExcepti
     protected int getImapPort() {
         return imapServer.getRemoteImapBindPort();
     }
+
+    @Override
+    protected void runOp(MailboxOperation op, ZMailbox mbox, ZFolder folder)
+            throws Exception {
+        op.run(mbox);
+        AssertionError saved = null;
+        int timeout = 6000;
+        while(timeout > 0) {
+            try {
+                op.checkResult();
+                return;
+            } catch (AssertionError e) {
+                saved = e;
+                Thread.sleep(500);
```

Comment:
I don't think my comments in the previous pull request have been addressed.  The fact that you are needing to introduce waits to get these tests to pass suggests one of two things to me:
* The tests are a waste of time because they are testing something that isn't relevant to the IMAP daemon.
* The IMAP daemon is fundamentally broken because it doesn't have access to notifications it needs to have access to and therefore will not function correctly as an IMAP server.
Either way, I don't see this pull request being useful.